src/Integration/Suppression/SonarQubeIssuesProvider.cs

```diff
@@ -204,6 +204,7 @@ private async Task SynchronizeSuppressedIssuesAsync()
                     .ToDictionary(x => x.Key, x => x.RelativePathToRoot);
                 this.hasModules = moduleKeyToRelativePathToRoot.Keys.Count > 1;
                 this.allSuppressedIssues = await this.sonarQubeService.GetSuppressedIssuesAsync(sonarQubeProjectKey,
+                    null, // TODO - pass the current branch
```

Comment:
For now I've just passed `null` for the branch in the product. I've manually tested that it still works.